Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM openjdk:8-jre
```

Comment:
Should we build our own base image instead?

Reply:
My company just started migration from Java 6 to Java 7, our code works and tests pass on 7 but we won't be deploying Java7 any time soon on production. Could you please change it to Java 6 and use Oracle Java, so more companies worldwide can benefit from your contribution?